### PR TITLE
doc(ecopasskit): add digital product passport introduction

### PR DESCRIPTION
**Pull request by FabianKentsch**
## Description

This PR adds more information to the EcoPassKit regarding Digital Product Passports 

Fixes https://github.com/eclipse-tractusx/sig-release/issues/649

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [x] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [x] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

**Review by jSchuetz88 — APPROVED** [review]
Thank you for the contribution, looks good to me.

**Review by danielmiehle — DISMISSED** [review]
LGTM

**Review by matbmoser — APPROVED** [review]
Ok this looks fine to me

File: docs-kits/kits/Eco_Pass_KIT/page-adoption-view.md [hunk 1]
```diff
@@ -2062,4 +2066,5 @@ This work is licensed under the [CC-BY-4.0](https://creativecommons.org/licenses
 - SPDX-FileCopyrightText: 2023, 2024 BASF SE
 - SPDX-FileCopyrightText: 2023, 2024 Henkel AG & Co. KGaA
 - SPDX-FileCopyrightText: 2023, 2024 Contributors to the Eclipse Foundation
+- SPDX-FileCopyrightText: 2023, 2024 MHP Management- und IT-Beratung GmbH (On behalf of Volkswagen AG)
```
Comment (catroest) [inline docs-kits/kits/Eco_Pass_KIT/page-adoption-view.md:2069]: how comes VW / MHP is popping-up here?

Comment (matbmoser) [reply docs-kits/kits/Eco_Pass_KIT/page-adoption-view.md:2069]: Did they participated in the review?

Comment (FabianKentsch) [reply docs-kits/kits/Eco_Pass_KIT/page-adoption-view.md:2069]: Actually, I am from MHP, and I participate in the expert group (and also wrote the whitpaper) on behalf of Volkswagen AG.

Comment (catroest) [reply docs-kits/kits/Eco_Pass_KIT/page-adoption-view.md:2069]: Ah, I am sorry, I am only used to the consortia team

**Review by matbmoser — CHANGES_REQUESTED** [review]
Please lets check the final notice IP checks

File: docs-kits/kits/Eco_Pass_KIT/page-adoption-view.md (hunk 1 above)
Comment (matbmoser) [reply docs-kits/kits/Eco_Pass_KIT/page-adoption-view.md:2069]: Yep but this on behalf is not correct

Comment (matbmoser) [reply docs-kits/kits/Eco_Pass_KIT/page-adoption-view.md:2069]: Add or both companies, or none

Comment (matbmoser) [reply docs-kits/kits/Eco_Pass_KIT/page-adoption-view.md:2069]: @FabianKentsch if you are working for Volkswagen AG then volkswagen needs to be included.

And they should clarify if MHP shall be included. Example: CGI was working for BMW, and BMW Allowed CGI to be listed.

**Review by matbmoser — APPROVED** [review]

**Review by matbmoser — CHANGES_REQUESTED** [review]
I will change it

**Comment by matbmoser** [comment]
Ok I could not change it because I have no permissions to commit in the fork
Please add the following licence note:

```md
## NOTICE

This work is licensed under the [CC-BY-4.0](https://creativecommons.org/licenses/by/4.0/legalcode).

- SPDX-License-Identifier: CC-BY-4.0
- SPDX-FileCopyrightText: 2023, 2024 ZF Friedrichshafen AG
- SPDX-FileCopyrightText: 2023, 2024 Robert Bosch GmbH
- SPDX-FileCopyrightText: 2023, 2024 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)
- SPDX-FileCopyrightText: 2023, 2024 T-Systems International GmbH
- SPDX-FileCopyrightText: 2023, 2024 SAP SE
- SPDX-FileCopyrightText: 2023, 2024 CGI Deutschland B.V. & Co. KG
- SPDX-FileCopyrightText: 2023, 2024 Fraunhofer-Gesellschaft zur Förderung der angewandten Forschung e.V. für ihre Institute IPK und IPK
- SPDX-FileCopyrightText: 2023, 2024 BASF SE
- SPDX-FileCopyrightText: 2023, 2024 Henkel AG & Co. KGaA
- SPDX-FileCopyrightText: 2024 Volkswagen AG
- SPDX-FileCopyrightText: 2024 MHP Management- und IT-Beratung GmbH
- SPDX-FileCopyrightText: 2023, 2024 Contributors to the Eclipse Foundation
- Source URL: <https://github.com/eclipse-tractusx/digital-product-pass>

```

**Review by matbmoser — APPROVED** [review]
LGTM

**Review by saudkhan116 — APPROVED** [review]
LGTM

**Comment by matbmoser** [comment]
I will merge this :) Wonderfull job everyone!

**Pull request merged**